api/v1beta1/nodemodulestate_types.go

```diff
@@ -20,16 +20,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type ModuleNodeSpec struct {
-	Name           string `json:"name"`
-	Namespace      string `json:"namespace"`
+type ModuleConfig struct {
```

Comment:
Shouldn't this file be in `api/v2/...`?

Reply:
i would prefer to leave it for now in v1. The official API to customer is not changing, so i am not sure if we want to move it to v2